### DISCUSSION TITLE
Call to a member function entryBlueprints() on string

### DISCUSSION THREAD

**Discussion by fienta-public** [statamic/cms#4922] [Troubleshooting] [answered]
Hi,

I am attempting to install statamic side by side with our existing laravel application.
Versions: statamic 3.2.26, laravel 6.20.43, php 7.3.33. 

In the control panel when I add a collection and get redirected to view it, an error occurs:
**Call to a member function entryBlueprints() on string** in vendor\statamic\cms\src\Http\Controllers\CP\Collections\CollectionsController.php:56
`$blueprints = $collection->entryBlueprints()`

I've tried doing dd($collection) at CollectionsController and what I get as a result isis "pages".

Same issue for viewing taxonomy after creating it:
**Call to a member function termBlueprints() on string** in vendor\statamic\cms\src\Http\Controllers\CP\Taxonomies\TaxonomiesController.php:54
`$blueprints = $taxonomy->termBlueprints()`

When trying to create a blueprint for a collection:
**Call to a member function handle() on string** in vendor\statamic\cms\src\Http\Controllers\CP\Collections\CollectionBlueprintsController.php:72
`if (Blueprint::in('collections/'.$collection->handle())->count() === 0) {`

Does anyone know where to start looking for what is causing this?

**Comment by jasonvarga** [comment]
We have route model binding set up for `{collection}` and `{taxonomy}` that should have converted those strings into classes.

Have you added any route model binding that might be interfering with ours?

**Reply by fienta-public** [reply]
I don't have any models called collection or taxonomy but there are many routes with route model binding. For testing purposes I tried to remove all the application's defined routes but it didn't solve the error. 
I also have some bindings defined in RouteServiceProvider.php. However removing these does not help statamic to find {collection} either.

**Comment by fienta-public** [comment, marked as answer]
This has been solved by removing Route::singularResourceParameters(false); from AppServiceProvider.
